sdk/simulator-service/src/strategies/refinanceAnyPair/RefinanceLendingToLendingAnyPair.ts

```diff
@@ -65,11 +53,14 @@ export async function refinanceLendingToLendingAnyPair(
       type: SimulationSteps.Flashloan,
       inputs: {
         amount: flashloanAmount,
-        provider: FlashloanProvider.Maker,
+        provider:
+          flashloanAmount.token.symbol === 'DAI'
```

Comment:
would be better to have reusable constant for it

Reply:
We do have `CommonTokenSymbols`. I'll refactor it